src/SignaturePad.Forms.Shared/SignaturePadCanvasView.cs

```diff
@@ -7,7 +7,6 @@
 
 namespace SignaturePad.Forms
 {
-	[RenderWith (typeof (SignaturePadCanvasRenderer))]
```

Comment:
How does this break the renderer?

Reply:
The page is not rendered at all, and throw the exeption unable to cast to Xamarin.Forms.IRegistable

Reply:
But this problem should go away now since we have a WPF renderer?

Reply:
Yes, I have returned it